bug(tests): golang tests failing

**Describe the bug**
```
 make test
\033[1mRunning Go short tests\033[0m
	github.com/carverauto/serviceradar/cmd/agent		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/cmd/cli		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/cmd/consumers/db-event-writer		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/cmd/data-services		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/cmd/faker	2.655s	coverage: 2.5% of statements in ./...
	github.com/carverauto/serviceradar/cmd/tools/waitforport		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/internal/fastsum		coverage: 0.0% of statements
-test.shuffle 1770783817532828752
coverage: 7.7% of statements in ./...
panic: test timed out after 15s
	running tests:
		TestExecuteWithWasmHarness (3s)

goroutine 354 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:2682 +0x605
created by time.goFunc
	/usr/local/go/src/time/sleep.go:215 +0x45

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0002c8a80, {0x19dfef6, 0x1a}, 0x1a28420)
	/usr/local/go/src/testing/testing.go:2005 +0x9fe
testing.runTests.func1(0xc0002c8a80)
	/usr/local/go/src/testing/testing.go:2477 +0x86
testing.tRunner(0xc0002c8a80, 0xc0004b1ae0)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
testing.runTests(0xc000206600, {0x24d8fa0, 0x41, 0x41}, {0x2?, 0x4ba6c5?, 0x24fefa0?})
	/usr/local/go/src/testing/testing.go:2475 +0x96d
testing.(*M).Run(0xc00026f0e0)
	/usr/local/go/src/testing/testing.go:2337 +0xed5
main.main()
	_testmain.go:195 +0x165

goroutine 322 [chan receive]:
testing.(*T).Parallel(0xc00051e000)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestNewSNMPAgentService(0xc00051e000)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:47 +0x3f
testing.tRunner(0xc00051e000, 0x1a28438)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 324 [chan receive]:
testing.(*T).Parallel(0xc00051e380)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSysmonServiceName(0xc00051e380)
	/home/mfreeman/serviceradar/pkg/agent/sysmon_service_test.go:95 +0x3c
testing.tRunner(0xc00051e380, 0x1a28598)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 333 [chan receive]:
testing.(*T).Parallel(0xc00051f340)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestNewServerWithSweepConfig(0xc00051f340)
	/home/mfreeman/serviceradar/pkg/agent/server_test.go:106 +0x3f
testing.tRunner(0xc00051f340, 0x1a28448)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 338 [chan receive]:
testing.(*T).Parallel(0xc00051fc00)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSysmonServiceGetStatus_NotStarted(0xc00051fc00)
	/home/mfreeman/serviceradar/pkg/agent/sysmon_service_test.go:144 +0x3f
testing.tRunner(0xc00051fc00, 0x1a28588)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 337 [chan receive]:
testing.(*T).Parallel(0xc00051fa40)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestNewSysmonService(0xc00051fa40)
	/home/mfreeman/serviceradar/pkg/agent/sysmon_service_test.go:41 +0x3f
testing.tRunner(0xc00051fa40, 0x1a28450)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 234 [runnable]:
github.com/tetratelabs/wazero/internal/engine/wazevo/backend/isa/amd64.(*labelPosition).InstrRevIteratorNext(0xc0003a0398)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/isa/amd64/machine_regalloc.go:182 +0x1de
github.com/tetratelabs/wazero/internal/engine/wazevo/backend/regalloc.(*Allocator[...]).livenessAnalysis(0x1be4180, 0xc0004c4140)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/regalloc/regalloc.go:441 +0x787
github.com/tetratelabs/wazero/internal/engine/wazevo/backend/regalloc.(*Allocator[...]).DoAllocation(0x1be4180, 0xc0004c4140)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/regalloc/regalloc.go:359 +0xae
github.com/tetratelabs/wazero/internal/engine/wazevo/backend/isa/amd64.(*machine).RegAlloc(0xc00018ae08)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/isa/amd64/machine.go:298 +0x135
github.com/tetratelabs/wazero/internal/engine/wazevo/backend.(*compiler).RegAlloc(...)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/compiler.go:191
github.com/tetratelabs/wazero/internal/engine/wazevo/backend.(*compiler).Compile(0xc000281860, {0x1bb0740, 0xc0001b6310})
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/backend/compiler.go:170 +0x11a
github.com/tetratelabs/wazero/internal/engine/wazevo.(*engine).compileLocalWasmFunction(0xd80?, {0x1bb0740, 0xc0001b6310}, 0xc0001c1380, 0x2d, 0xc000216008, {0x1bc7c40, 0xc000a48008}, {0x1bc19d0, 0xc000281860}, ...)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/engine.go:515 +0x382
github.com/tetratelabs/wazero/internal/engine/wazevo.(*engine).compileModule(0xc000174360, {0x1bb0740, 0xc0001b6310}, 0xc0001c1380, {0x0, 0x0, 0x0}, 0x0)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/engine.go:265 +0xeca
github.com/tetratelabs/wazero/internal/engine/wazevo.(*engine).CompileModule(0xc000174360, {0x1bb0740, 0xc0001b6310}, 0xc0001c1380, {0x0, 0x0, 0x0}, 0x0)
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/internal/engine/wazevo/engine.go:149 +0x19e
github.com/tetratelabs/wazero.(*runtime).CompileModule(0xc000360660, {0x1bb0740, 0xc0001b6310}, {0xc0003c0000, 0x2830d, 0x2830e})
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/runtime.go:262 +0x562
github.com/tetratelabs/wazero.(*runtime).InstantiateWithConfig(0xc000360660, {0x1bb0740, 0xc0001b6310}, {0xc0003c0000, 0x2830d, 0x2830e}, {0x1bbed40, 0xc000d0a960})
	/home/mfreeman/serviceradar/.gomodcache/github.com/tetratelabs/wazero@v1.11.0/runtime.go:298 +0xe5
github.com/carverauto/serviceradar/pkg/agent.(*PluginManager).executeWithWasm(0xc00025a480, {0x1bb0740, 0xc0001b6310}, 0xc00025b080, {0xc0003c0000, 0x2830d, 0x2830e})
	/home/mfreeman/serviceradar/pkg/agent/plugin_runtime.go:981 +0x197b
github.com/carverauto/serviceradar/pkg/agent.TestExecuteWithWasmHarness(0xc00035ce00)
	/home/mfreeman/serviceradar/pkg/agent/plugin_runtime_local_test.go:65 +0xabc
testing.tRunner(0xc00035ce00, 0x1a28420)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 306 [chan receive]:
testing.(*T).Parallel(0xc0002c9a40)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSNMPAgentServiceName(0xc0002c9a40)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:114 +0x3c
testing.tRunner(0xc0002c9a40, 0x1a284d0)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 127 [chan receive]:
testing.(*T).Parallel(0xc0002c9500)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSNMPAgentServiceGetStatus_NotStarted(0xc0002c9500)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:217 +0x3f
testing.tRunner(0xc0002c9500, 0x1a284b8)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 311 [chan receive]:
testing.(*T).Parallel(0xc000188fc0)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSNMPAgentServiceConfigCaching(0xc000188fc0)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:339 +0x3f
testing.tRunner(0xc000188fc0, 0x1a284a8)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 328 [chan receive]:
testing.(*T).Parallel(0xc00051ea80)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSNMPAgentServiceStopIdempotent(0xc00051ea80)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:261 +0x3f
testing.tRunner(0xc00051ea80, 0x1a284e0)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3

goroutine 309 [chan receive]:
testing.(*T).Parallel(0xc0001888c0)
	/usr/local/go/src/testing/testing.go:1708 +0x50c
github.com/carverauto/serviceradar/pkg/agent.TestSNMPConfigHashComputation(0xc0001888c0)
	/home/mfreeman/serviceradar/pkg/agent/snmp_service_test.go:532 +0x3c
testing.tRunner(0xc0001888c0, 0x1a284e8)
	/usr/local/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/usr/local/go/src/testing/testing.go:1997 +0x9d3
FAIL	github.com/carverauto/serviceradar/pkg/agent	17.053s
ok  	github.com/carverauto/serviceradar/pkg/agent/core	2.246s	coverage: 0.5% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/agent/snmp	3.687s	coverage: 2.1% of statements in ./...
	github.com/carverauto/serviceradar/pkg/agentgateway		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/pkg/cli		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/config	3.415s	coverage: 1.6% of statements in ./...
	github.com/carverauto/serviceradar/pkg/config/bootstrap		coverage: 0.0% of statements
?   	github.com/carverauto/serviceradar/pkg/config/kv	[no test files]
	github.com/carverauto/serviceradar/pkg/config/kvgrpc		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/pkg/config/kvnats		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/consumers/db-event-writer	4.205s	coverage: 2.5% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/cpufreq	2.051s	coverage: 0.4% of statements in ./...
	github.com/carverauto/serviceradar/pkg/crypto/secrets		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/datasvc	5.741s	coverage: 1.9% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/db	3.901s	coverage: 1.7% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/devicealias	2.239s	coverage: 0.4% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/deviceupdate	3.677s	coverage: 1.5% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/edgeonboarding	3.926s	coverage: 1.7% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/edgeonboarding/mtls	3.811s	coverage: 1.6% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/grpc	3.955s	coverage: 2.8% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/hashutil	2.285s	coverage: 0.3% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/integration_test	3.744s	coverage: 1.7% of statements in ./...
	github.com/carverauto/serviceradar/pkg/lifecycle		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/logger	3.504s	coverage: 1.7% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/mapper	3.876s	coverage: 2.4% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/models	3.782s	coverage: 1.6% of statements in ./...
	github.com/carverauto/serviceradar/pkg/monitoring		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/nats/accounts	3.890s	coverage: 1.9% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/natsutil	3.582s	coverage: 1.4% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/scan	3.698s	coverage: 1.5% of statements in ./...
	github.com/carverauto/serviceradar/pkg/spireadmin		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/pkg/swagger		coverage: 0.0% of statements
ok  	github.com/carverauto/serviceradar/pkg/sweeper	4.181s	coverage: 2.6% of statements in ./...
ok  	github.com/carverauto/serviceradar/pkg/sysmon	2.856s	coverage: 2.0% of statements in ./...
	github.com/carverauto/serviceradar/pkg/version		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/proto		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/proto/discovery		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/proto/flow		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/proto/identitymap/v1		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/proto/rperf		coverage: 0.0% of statements
	github.com/carverauto/serviceradar/release		coverage: 0.0% of statements
FAIL
make: *** [Makefile:190: test] Error 1
➜  serviceradar git:(2782-bugcore-elx-otlp-grpc-channel-has-no-endpoints)
```

**To Reproduce**
Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/pkg/agent/plugin_runtime_local_test.go
+++ b/pkg/agent/plugin_runtime_local_test.go
@@ -18,6 +18,10 @@ import (
 )
 
 func TestExecuteWithWasmHarness(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping: WASM execution is too slow for short mode")
+	}
+
 	wasmPath := os.Getenv("WASM_PATH")
 	if wasmPath == "" {
 		_, thisFile, _, ok := runtime.Caller(0)

--- a/web-ng/mix.exs
+++ b/web-ng/mix.exs
@@ -21,7 +21,14 @@ defmodule ServiceRadarWebNG.MixProject do
   def application do
     [
       mod: {ServiceRadarWebNG.Application, []},
-      extra_applications: [:logger, :runtime_tools, :swoosh, :telemetry, :opentelemetry, :opentelemetry_experimental]
+      extra_applications: [
+        :logger,
+        :runtime_tools,
+        :swoosh,
+        :telemetry,
+        :opentelemetry,
+        :opentelemetry_experimental
+      ]
     ]
   end
 

